client/src/services/fileOperationManager.ts

```diff
@@ -808,6 +868,80 @@ class FileOperationManager {
     }
   }
 
+  private async doDownloadArchive(data: DownloadArchiveData): Promise<void> {
```

Comment:
The generated zip file is incompatible with MacOS 
<img width="490" height="246" alt="Screenshot 2025-07-29 at 10 29 06" src="https://github.com/user-attachments/assets/ac4ffe0b-32d4-4a3d-8080-9918dab54b4e" />